README.md

```diff
@@ -1,2 +1,75 @@
-# RACTF Backend
-The backend API for the RACTF platform.
+<h1 align="center">RACTF Core</h1>
+
+<div align="center">
+  🚩⛳🏁🏳️‍🌈
```

Comment:
:rainbow_flag: :+1: 

Not sure how useful the others are

Reply:
goal is to include as many flag emojis as possible

Reply:
Maybe the space would be better served with an RACTF logo?

Reply:
emoji add personality and flair

Reply:
Branding adds a recognisable logo